src/main.cc

```diff
@@ -1445,9 +1447,13 @@ Parse(TRITONSERVER_ServerOptions** server_options, int argc, char** argv)
         exit_on_error = ParseBoolOption(optarg);
         break;
       case OPTION_DISABLE_AUTO_COMPLETE_CONFIG:
-        strict_model_config = true;
+        disable_auto_complete_config = true;
         break;
       case OPTION_STRICT_MODEL_CONFIG:
+        std::cerr << "Warning: '--strict-model-config' has been deprecated! "
```

Comment:
Use `LOG_MESSAGE(TRITONSERVER_LOG_WARN, "message")` instead of std::cerr.

Reply:
LOG_MESSAGE hasn't been declared in this scope and I am not sure it should be since we allow logging to be turned on and off at compile time. I was copying this pattern from `OPTION_MODEL_CONTROL_MODE` or some of the other case statements.